app/src/main/java/hm/orz/chaos114/android/tumekyouen/util/EncryptionUtil.java

```diff
@@ -8,108 +11,102 @@
 import javax.crypto.KeyGenerator;
 import javax.crypto.spec.SecretKeySpec;
 
-import android.content.Context;
-import android.util.Base64;
-
 public class EncryptionUtil {
-	/** 鍵のbit数 */
-	private static final int ENCRYPT_KEY_LENGTH = 128;
+    /** 鍵のbit数 */
+    private static final int ENCRYPT_KEY_LENGTH = 128;
+
+    /** 暗号化キー */
+    private final Key key;
 
-	/** 暗号化キー */
-	private final Key key;
+    /**
+     * コンストラクタ。
+     * 暗号化キーを生成、もしくは復元します。
+     *
+     * @param context コンテキスト
+     */
+    public EncryptionUtil(Context context) {
+        context = context.getApplicationContext();
 
-	/** コンテキスト */
-	private final Context context;
+        // Preferenceから暗号化キーを取得
+        PreferenceUtil preferenceUtil = new PreferenceUtil(context);
+        String keyStr = preferenceUtil.getString(PreferenceUtil.KEY_SECRET_KEY);
 
-	/**
-	 * コンストラクタ。
-	 * 暗号化キーを生成、もしくは復元します。
-	 * 
-	 * @param context コンテキスト
-	 */
-	public EncryptionUtil(Context context) {
-		this.context = context.getApplicationContext();
-		
-		// Preferenceから暗号化キーを取得
-		PreferenceUtil preferenceUtil = new PreferenceUtil(this.context);
-		String keyStr = preferenceUtil.getString(PreferenceUtil.KEY_SECRET_KEY);
+        if (keyStr == null) {
+            // Preferenceから取得できなかった場合
 
-		if (keyStr == null) {
-			// Preferenceから取得できなかった場合
+            // 暗号化キーを生成
+            key = generateKey();
+            // 生成したキーを保存
+            String base64Key = Base64.encodeToString(key.getEncoded(),
+                    Base64.URL_SAFE | Base64.NO_WRAP);
+            preferenceUtil.putString(PreferenceUtil.KEY_SECRET_KEY, base64Key);
+        } else {
+            // Preferenceから取得できた場合
 
-			// 暗号化キーを生成
-			key = generateKey();
-			// 生成したキーを保存
-			String base64Key = Base64.encodeToString(key.getEncoded(),
-					Base64.URL_SAFE | Base64.NO_WRAP);
-			preferenceUtil.putString(PreferenceUtil.KEY_SECRET_KEY, base64Key);
-		} else {
-			// Preferenceから取得できた場合
-			
-			// キーを復元
-			byte[] keyBytes = Base64.decode(keyStr, Base64.URL_SAFE
-					| Base64.NO_WRAP);
-			key = new SecretKeySpec(keyBytes, "AES");
-		}
-	}
+            // キーを復元
+            byte[] keyBytes = Base64.decode(keyStr, Base64.URL_SAFE
+                    | Base64.NO_WRAP);
+            key = new SecretKeySpec(keyBytes, "AES");
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "AES" appears 4 times in this file; the first occurrence is on line 49
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals